### PR TITLE
Changed HTTP library, updated tests

### PR DESCRIPTION
**Pull request by VitaliyRDev**
Changed HTTP library from 'request' to 'got', updated tests, also implemented new throttle function for the compatibility with the 'got' library.

### CODE REVIEW

File: lib/utils/sleep.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = function (ms) {
```
Comment (facundoolano) [inline lib/utils/sleep.js:1]: a module for this is overkill, just duplicate the function in the two modules that use it

File: lib/utils/throttle.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+const sleep = require('./sleep.js');
+
+function Throttle () {
```
Comment (facundoolano) [inline lib/utils/throttle.js:3]: is this your code or you copied or adapted it from somewhere else?

Comment (VitaliyRDev) [reply lib/utils/throttle.js:3]: Throttle function I've written by myself, sleep function is copied

Comment (facundoolano) [reply lib/utils/throttle.js:5]: Good, in that case, can you add a docstring explaining how this works (e.g. expected settings) and some basic unit tests showing that it works?

File: lib/utils/sleep.js (hunk 1 above)
Comment (VitaliyRDev) [reply lib/utils/sleep.js]: > 

Done.

File: lib/utils/throttle.js (hunk 2 above)
Comment (VitaliyRDev) [reply lib/utils/throttle.js:5]: Sure, but it will take some time.

Comment (facundoolano) [reply lib/utils/throttle.js:5]: I have no hurry, thank you for the contribution.

**Comment by VitaliyRDev** [comment]
@facundoolano waiting for review...

**Comment by facundoolano** [comment]
Thanks!

**Pull request merged**